src/concat-benchmark.cc

```diff
@@ -89,6 +93,25 @@ void format_to(benchmark::State& state) {
   }
 }
 BENCHMARK(format_to);
+#if __cpp_lib_concepts>=201907L
+void fast_io_print(benchmark::State& state) {
+  benchmark::ClobberMemory();
+  for (auto _ : state) {
+    fast_io::internal_temporary_buffer<char> output;
+    print(output, "Result: ",str1,": (",str2,",",str3,",",str4,",",str5,")");
+    benchmark::DoNotOptimize(obuffer_begin(output));
+  }
+}
+BENCHMARK(fast_io_print);
```

Comment:
This doesn't make much sense because the goal of this benchmark is to produce `std::string`.

Reply:
Then remove this